armada/handlers/wait_test.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+from armada import Armada
+import tiller
+from k8s import K8s
+
+import mock
+import unittest
+
+class WaitTestCase(unittest.TestCase):
+
+    @mock.patch('armada.handlers.tiller.Tiller')
+    def test_wait(self, mock_tiller):
+        # instantiate Armada object
+        armada = Armada("../../examples/openstack-helm.yaml")
+        armada.tiller = mock_tiller
+
+        # assemble test pod info to give to the wait function
+        pods = mock.Mock()
+        pods.items = [mock.Mock()]
+        pods.items[0].metadata.name = 'test_pod_1'
+        pods.items[0].status.conditions = [mock.Mock()]
+        pods.items[0].status.conditions[0].status = 'False'
+        pods.items[0].status.conditions[0].message = 'intentional_fail'
+
+        mock_tiller.k8s.get_all_pods.return_value = pods
+#         {
+#             'items': 
+#                 [
```

Comment:
is this because you want to leave it as part of the documentation or is this because you are commenting out code?